akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpMessage.scala

```diff
@@ -67,9 +67,6 @@ sealed trait HttpMessage extends jm.HttpMessage {
   /** Returns a copy of this message with the list of headers transformed by the given function */
   def mapHeaders(f: immutable.Seq[HttpHeader] ⇒ immutable.Seq[HttpHeader]): Self = withHeaders(f(headers))
 
-  /** Returns a copy of this message with the entity transformed by the given function */
-  def mapEntity(f: HttpEntity ⇒ MessageEntity): Self = withEntity(f(entity))
```

Comment:
Because it clashes with the `mapEntity` methods on `HttpRequest` and `HttpResponse` and wasn't usable anyway.
Actually I wasn't even sure how things compiled here in the first place.